plugin/02tlib.vim

```diff
@@ -74,7 +74,7 @@ command! -nargs=1 -complete=command TBrowseOutput call tlib#cmd#BrowseOutput(<q-
 "
 " EXAMPLES: >
 "   TBrowseScriptnames 
-command! -nargs=0 -complete=command TBrowseScriptnames call tlib#cmd#TBrowseScriptnames()
+command! -nargs=0 TBrowseScriptnames call tlib#cmd#TBrowseScriptnames()
```

Comment:
I've seen an alternative fix that looks like this:
```suggestion
command! -nargs=? -complete=command TBrowseScriptnames call tlib#cmd#TBrowseScriptnames()
```

...but I don't really understand the difference. Is there any advantage of one over the other?

Reply:
I am hitting this error as well, and I think dropping the `-complete=command` entirely is the correct solution. `-complete` specifies the type of completion that the command arguments would undergo. In this case the function takes no arguments so there's nothing to complete. In your example the `?` means 0 or 1 arguments, so there could be an argument to complete.

Reply:
Thanks for the explanation!